test/test.js

```diff
@@ -101,21 +101,21 @@ test("minify with sourcemaps", async () => {
     plugins: [terser()],
   });
   const result = await bundle.generate({ format: "cjs", sourcemap: true });
-  expect(result.output).toHaveLength(1);
+  expect(result.output).toHaveLength(2);
```

Comment:
I don't know or understand why this became `2`.
Can someone verify that this is ok?